target/linux/ath79/image/generic.mk

```diff
@@ -1071,3 +1071,12 @@ define Device/zbtlink_zbt-wd323
 		     kmod-usb-serial kmod-usb-serial-cp210x uqmi
 endef
 TARGET_DEVICES += zbtlink_zbt-wd323
+
+define Device/zsun_sd100
+  ATH_SOC := ar9331
```

Comment:
Please rename this to "SOC" due to 95030ac0b673165772daf27128359eaaf536d457.

Reply:
Thanks for the heads up, it's now renamed.
Wish you a Merry Christmas and a Happy New Year!